silence/silence_test.go

```diff
@@ -547,6 +549,33 @@ func TestSilenceLimits(t *testing.T) {
 	sil6, err = s.QueryOne(QIDs(sil6.Id))
 	require.NoError(t, err)
 	require.Equal(t, types.SilenceStateActive, getState(sil6, s.nowUTC()))
+
+	// Should not be able to update with a comment that exceeds maximum size.
```

Comment:
These are the additional test cases that make sure an existing silence is not expired when the `MaxSilenceSizeBytes` limit is exceeded.